docs/src/_guide/configuration.md

```diff
@@ -127,12 +185,14 @@ When Meltano invokes a plugin's executable as part of [`meltano elt`](/reference
 These can then be accessed from inside the plugin using the mechanism provided by the standard library, e.g. Python's [`os.environ`](https://docs.python.org/3/library/os.html#os.environ).
 
 Within a [Meltano environment](/concepts/environments) environment variables can be specified using the `env` key:
+
 ```yml
 environments:
   - name: dev
     env:
       AN_ENVIRONMENT_VARIABLE: dev
 ```
+
 Any plugins run in that Meltano environment will then have the provided environment variables populated into the plugin's environment.
 
 ## Multiple plugin configurations
```

Comment:
There's a yml section just below this that has highlighting of specific rows using `{8-18}`. Can we remove that here too since it currently doesn't work?

Reply:
I made https://github.com/meltano/meltano/issues/6287 to track adding that feature back.